crates/topos-tce/src/app_context/api.rs

```diff
@@ -29,6 +30,16 @@ impl AppContext {
                             certificate.id, certificate.source_subnet_id
                         );
 
+                        _ = self
+                            .tce_cli
+                            .get_double_echo_channel()
+                            .send(DoubleEchoCommand::Broadcast {
```

Comment:
Why are we doing this? And why is it needed now but wasn't before?

Reply:
See description